subseas_flex_fcst/layout.py

```diff
@@ -278,15 +278,15 @@ def navbar_layout(phys_units):
             html.Div(
                 [
                     help_layout(
-                        "Lead time",
+                        "Target Date",
                         "lead_time",
-                        "Lead time range choices, in weeks from the start date",
+                        "Target date ranges calculated using lead times and model starts.",
```

Comment:
Also this might have a better description than I wrote..

Reply:
Maybe: "Time period being forecasted"
Maproom text can elaborate on the relationship between S/L/target if need be.